lib/main.dart

```diff
@@ -108,3 +139,22 @@ class QuimifyApp extends StatelessWidget {
     );
   }
 }
+
+Future<void> configurePurchaseSDK() async {
+  await Purchases.setLogLevel(LogLevel.debug);
+
+  PurchasesConfiguration? configuration;
+
+  if (Platform.isAndroid) {
+    configuration = PurchasesConfiguration('goog_SgCLLuxoLOuSMTDVBqzacRsJFaD');
```

Comment:
Are those values secret?

Reply:
These are public api keys.
You can find details here:

https://www.revenuecat.com/docs/welcome/authentication

Reply:
If you want you can keep these in the env, but I don't have access to all the API keys only the env.g.dart file so can't regenerate and risk losing stuff, so best you guys do the generating of these keys